test/coverage/datamodel/UUriValidatorTest.cpp

```diff
@@ -123,41 +124,50 @@ TEST_F(TestUUriValidator, Wildcards) {  // NOLINT
 
 	{  // Check for no wildcards
 		auto uuri = get_u_uri();
-		EXPECT_FALSE(uses_wildcards(uuri));
+		EXPECT_FALSE(has_wildcard_authority(uuri));
+		EXPECT_FALSE(has_wildcard_service_id(uuri));
+		EXPECT_FALSE(has_wildcard_service_instance_id(uuri));
+		EXPECT_FALSE(has_wildcard_version(uuri));
+		EXPECT_FALSE(has_wildcard_resource_id(uuri));
+		EXPECT_TRUE(verify_no_wildcards(uuri));
 	}
 
 	{  // Change Authority name to "hello*" (Any)
```

Comment:
Update comment here, right?
```suggestion
	{  // Change Authority name to "*" (Any)
```